itu-minitwit-golang/internal/utils/create_user.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"fmt"
+	"itu-minitwit/internal/models"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"golang.org/x/crypto/bcrypt"
+	"gorm.io/gorm"
+)
+
+// Creates a user and adds the user to the database.
+// It returns a boolean value, that determines if the value insertion is done correctly
+func CreateUser(c *gin.Context, username string, email string, password string) {
```

Comment:
The gin context should really not be needed here, instead return an error if something goes wrong